locale/ca/index.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: index.hbs
+labels:
+  current-version: Versió actual
+  download: Descarregar
+  download-for: Descarregar per
+  other-downloads: Altres Descàrregues
+  other-lts-downloads: Altres Descàrregues LTS
+  other-current-downloads: Altres Descàrregues Actuals
+  current: Actual
+  lts: LTS
+  tagline-current: Últimes característiques
+  tagline-lts: Recomanat per a la majoria
+  changelog: Canvis
+  api: Documentació de l'API
+  version-schedule-prompt: O revisi la
+  version-schedule-prompt-link-text: Agenda de LTS
+---
+
+Node.js® és un entorn d'execució per a JavaScript construït amb el [motor de JavaScript V8 de Chrome](https://developers.google.com/v8/).
+Node.js fa servir un model d'operacions E/S sense bloqueig i orientat a esdeveniments, que el fa lleuger i eficient.
```

Comment:
thank you @sgimeno I just committed changes according your indications https://github.com/nodejs/nodejs.org/pull/1221/commits/6b1a8b22960a6b245ebba5fe46d6c6f699d8475b 

If everything looks ok for you, could you give me :+1: ?